packages/@react-aria/utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-aria/utils",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
when we publish with lerna, this'll get updated, so no need to do it in PR

Reply:
perfect.  I couldn't tell it this project auto-versioned or not.